### PR TITLE
Edit/Issue-#575: Update Design Guide figure image and downloadable pdf

### PR DESCRIPTION
**Pull request by jenpark-getty**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [ ] I am requesting feedback on a draft pull request

### Please describe the goal of this pull request and the changes that were made.

Added updated Guide to the quire-docs repo, being sure to update both the figure image version and the downloadable PDF

### CODE REVIEW

**Comment by Erin-Cecele** [comment]
looks great, @jenpark-getty!

**Review by Erin-Cecele — APPROVED** [review]

**Comment by jenpark-getty** [comment]
I added one more commit titled update cheatsheet.md addressing the task below, please take a look!
-
[ ] Edit the Markdown page. In the YAML section there is a reference to weight: that should be order:, and to the data folder that is now _data. And then in the Shortcodes section there's a lot that needs updating. Take your best stab at it and then @Erin-Cecele or I can make any comments/edits when we review your pull request.

File: content/resources/cheatsheet.md [hunk 1]
```diff
@@ -94,17 +94,19 @@ figure_list:
 
 ## Shortcodes
 
-A shortcode is a snippet of code inserted in a content (`.md`) file that pulls in information from other files in your project. For example, in Quire, the `q-figure` shortcode is used to insert an image from the `static/img` folder. It additionally references the `id`, `caption`, and other optional information from your `figures.yaml` data file. Other shortcodes include `q-cite`, `q-bibliography`, and `q-contributor.` Examples of proper formatting for these shortcodes are below.
+A shortcode is a snippet of code inserted in a content (`.md`) file that pulls in information from other files in your project. For example, in Quire, the `% figure %` shortcode is used to insert an image from the `content/_assets/images` folder. It additionally references the `id`, `caption`, and other optional information from your `figures.yaml` data file. Other shortcodes include `% cite %`, `% contributors %`, and `% accordion %`. Examples of proper formatting for these shortcodes are below.
```
Comment (Erin-Cecele) [inline content/resources/cheatsheet.md:97]: Please correct shortcode references to {% figure %}, {% cite %}, etc. Thank you!

**Review by Erin-Cecele — CHANGES_REQUESTED** [review]
Great work. Can you make the fix requested below and then also remove `online: false` from the page YAML. I want to preview the page before approving. Thanks so much, @jenpark-getty.

**Comment by jenpark-getty** [comment]
Hey @Erin-Cecele there doesn't seem to be an online: false line in the YAML, is there another part of the file I can edit for you?

Screenshot 2023-09-12 at 2 50 23 PM

**Comment by Erin-Cecele** [comment]
@jenpark-getty Oops, apologies. I meant these two fields: 

```
menu: false
toc: false
```

**Comment by jenpark-getty** [comment]
> @jenpark-getty Oops, apologies. I meant these two fields:
> 
> ```
> menu: false
> toc: false
> ```

Done!

**Comment by Erin-Cecele** [comment]
Hi @jenpark-getty, everything looks good. But I'm wondering if it might be useful to point people to the parts of the documentation that elaborate on each of these items, most specifically for the shortcodes. That could be hyperlinking the shortcode to the page (`{% figure %}` would link to https://quire.getty.edu/docs-v1/figure-images/) or you could more explicitly point people to other parts of the docs ("for more information see..."). What do you think?

**Comment by jenpark-getty** [comment]
That's a good idea, I'll get to it!

**Review by Erin-Cecele — APPROVED** [review]
Looks great. Thanks for your work on this @jenpark-getty

**Pull request merged**